src/modules/repository/repository-module-manager.js

```diff
@@ -469,23 +466,21 @@ class RepositoryModuleManager extends BaseModuleManager {
     }
 
     async getParanetSyncedAssetRecordByUAL(ual, options = {}) {
-        return this.getRepository('paranet_synced_asset').getParanetSyncedAssetRecordByUAL(
-            ual,
-            options,
-        );
+        return this.getRepository('paranet_asset').getParanetSyncedAssetRecordByUAL(ual, options);
     }
 
-    async getParanetSyncedAssetRecordsCountByDataSource(paranetUal, dataSource, options = {}) {
+    async getParanetSyncedAssetRecordsCountByDataSource(paranetUal, options = {}) {
```

Comment:
Why did you change old function?

Reply:
we dont need this one at all, can remove it completely